### PR TITLE
Revised copy for 'Description' tab

### PR DESCRIPTION
**Pull request by MarinaWoo**
Warren, Nicole and I have worked on a revised blurb for the description page. It was mostly Nicole, I just did some little tweaks.

Copy in this doc: https://docs.google.com/document/d/1NhaPzGg9I_Esf3N_YqH7ox9s0PiEilEgni_a4DlGqpw/edit

I have edited the existing 'description' as per the google doc, and then added links (as there were none in the doc) for the various extensions mentioned, and a couple of other things that made sense to have links.

Over to you :)

### CODE REVIEW

**Comment by mikejolley** [comment]
@mattyza Is this still accurate:

> WooCommerce is audited by Sucuri, the industry leader in WordPress plugin safety and security.

It *was* audited, but its not ongoing?

**Comment by mattyza** [comment]
@mikejolley @MarinaWoo I'd remove the Sucuri item, or reword it to say that Sucuri have reviewed WooCommerce previously.

There are talks of ongoing developer reviews, yet those talks have quietened down for a bit, it seems.

**Comment by MarinaWoo** [comment]
@mikejolley good spot.
